LR_Tourist/BLL/Services/TravelVoucherService.cs

```diff
@@ -0,0 +1,150 @@
+                Arrival = travelVoucher.Arrival,
+                Departure = travelVoucher.Departure,
+                Price = travelVoucher.Price,
+                ServicesId = travelVoucher.ServicesId,
+                HotelId = travelVoucher.HotelId,
+                StaffId = travelVoucher.StaffId,
+                UserId = travelVoucher.Id,
+            };
+        }
+
+        public async Task<IEnumerable<TravelVoucher>> GetTravelVouchers()
+        {
+            return _mapper.Map<IEnumerable<TravelVoucherDTO>, List<TravelVoucher>>(await repoTravelVoucher.GetAll());
+        }
+
+        public async Task MakeOrder(int idTravelVoucher,int idUser)
+        {
+            var user = await repoUser.Get(idUser);
+            var travelVoucher = await repoTravelVoucher.Get(idTravelVoucher);
+            if (user == null)
+            {
+                throw new ArgumentException("user not found");
+            }
+            else if(travelVoucher == null)
+            {
+                throw new ArgumentException("Travel voucher not found", "");
+            }
+            travelVoucher.UserId = user.Id;
+            await Update(_mapper.Map <TravelVoucher>(travelVoucher));
+        }
+
+        public async Task Create(TravelVoucher item)
+        {
+            if (item == null)
+            {
+                throw new ArgumentNullException(nameof(item));
+            }
+            else
+            {
+                if (CheckId(item.StaffId, item.HotelId,item.ServicesId,item.UserId))
+                {
+                    throw new ArgumentException("Check id");
+                }
+                else
+                {
+                    await repoTravelVoucher.Create(_mapper.Map<TravelVoucherDTO>(item));
+                }
+            }
+        }
+
+        public bool CheckId(int idStaff,int idHotel,int idService,int idUser)
+        {
+            try
+            {
+                
+                var staff = repoStaff.Get(idStaff);
+                var service = repoService.Get(idService);
+                var hotel = repoHotel.Get(idHotel);
+                var user = repoUser.Get(idUser);
+                if(staff == null || service == null || hotel == null)
+                {
+                    return true;
+                }
+                return false;
+            }
+            catch
+            {
+                throw new ArgumentException("Check id");
+            }
+        }
+
+        public async Task Update(Model.TravelVoucher item)
+        {
+            if (item == null)
+            {
+                throw new ArgumentNullException(nameof(item));
+            }
+            else
+            {
+                if (CheckId(item.StaffId, item.HotelId, item.ServicesId, item.UserId))
+                {
+                    throw new ArgumentException("Check id");
+                }
+                await repoTravelVoucher.Update(_mapper.Map<DA.Data.TravelVoucherDTO>(item));
```

Comment:
The useless namespace for type.
Clean this file.

Also the same situation with `Model.TravelVoucher` type.